packages/components/app/styles/components/card/container.scss

```diff
@@ -6,7 +6,10 @@
 //
 
 
+$hds-card-container-style: ( 'surface', 'elevation' );
 $hds-card-container-levels: ( 'base', 'mid', 'high' );
+$hds-card-container-hover-levels: ( 'base', 'mid', 'high' );
+$hds-card-container-active-levels: ( 'base', 'mid', 'high' );
```

Comment:
Same question here about redefining the levels for each state.

Reply:
See above.

Reply:
As per @heatherlarsen feedback, for now we keep the same three levels for all of them, so I have unified the arrays in a single one.